### PR TITLE
docs: update Multisend pattern example

### PR DESCRIPTION
**Pull request by solarthesis**
### What I did

fixes: #

The example of setting up a multisend safe transaction

### How I did it

replacing `Transaction()` with `MultiSend()`

### How to verify it

NA

### Checklist

- [X] Passes all linting checks (pre-commit and CI jobs)
~~- [ ] New test cases have been added and are passing~~
- [X] Documentation has been updated
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,15 +132,15 @@ from ape_safe import multisend
 from ape import accounts
 from ape_tokens import tokens
 
-safe = accounts.load("my-safe")
+safe   = accounts.load("my-safe")
```
Comment (fubuloubu) [inline README.md:135]:
```suggestion
safe = accounts.load("my-safe")
```

**Review by fubuloubu — APPROVED** [review]

**Comment by solarthesis** [comment]
hey @fubuloubu not sure if worth a shoutout in the docs here, happy to update, but `gas=0 ` needs to be passed if staging multi-send on L2, otherwise the staging would revert with the below
```
TransactionError: Gas estimation failed: 'gas required exceeds allowance (0)'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.
```

**Comment by fubuloubu** [comment]
> hey @fubuloubu not sure if worth a shoutout in the docs here, happy to update, but `gas=0 ` needs to be passed if staging multi-send on L2, otherwise the staging would revert with the below
> 
> ```
> TransactionError: Gas estimation failed: 'gas required exceeds allowance (0)'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.
> ```

Yes please add! We haven't tried to test against L2 implementation yet. This is for the one you tried on Optimism?

**Comment by fubuloubu** [comment]
> > > hey @fubuloubu not sure if worth a shoutout in the docs here, happy to update, but `gas=0 ` needs to be passed if staging multi-send on L2, otherwise the staging would revert with the below
> > > ```
> > > TransactionError: Gas estimation failed: 'gas required exceeds allowance (0)'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.
> > > ```
> > 
> > 
> > Yes please add! We haven't tried to test against L2 implementation yet. This is for the one you tried on Optimism?
> 
> yes sir, exactly, will add, but worried about repercussions on L1 if someone passes `gas=0` on L1 staging

If you can add the docs you are thinking that would be necessary to show this, it would help me comment on them so we can move forwards and merge this :fire:

**Comment by solarthesis** [comment]
> > > > hey @fubuloubu not sure if worth a shoutout in the docs here, happy to update, but `gas=0 ` needs to be passed if staging multi-send on L2, otherwise the staging would revert with the below
> > > > ```
> > > > TransactionError: Gas estimation failed: 'gas required exceeds allowance (0)'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.
> > > > ```
> > > 
> > > 
> > > Yes please add! We haven't tried to test against L2 implementation yet. This is for the one you tried on Optimism?
> > 
> > 
> > yes sir, exactly, will add, but worried about repercussions on L1 if someone passes `gas=0` on L1 staging
> 
> If you can add the docs you are thinking that would be necessary to show this, it would help me comment on them so we can move forwards and merge this 🔥

sure thing will do it a bit later today, no problem

**Comment by kaleb-keny** [comment]
hey @fubuloubu @solarthesis is helping me out with using this repo and i've updated the comments based after discussing with him

File: ape_safe/multisend.py [hunk 2]
```diff
@@ -90,13 +90,21 @@ class MultiSend(ManagerAccessMixin):
     Usage example::
 
         from ape_safe import multisend
-
+        from ape import accounts
+        
+        #loads the safe
```
Comment (fubuloubu) [inline ape_safe/multisend.py:95]:
```suggestion
        # load the safe
```

File: ape_safe/multisend.py [hunk 3]
```diff
@@ -90,13 +90,21 @@ class MultiSend(ManagerAccessMixin):
...
+        
         txn = multisend.MultiSend()
         txn.add(contract.myMethod, *call_args)
         txn.add(contract.myMethod, *call_args)
         ...  # Add as many calls as desired to execute
         txn.add(contract.myMethod, *call_args)
-        receipt = txn(sender=my_signer)  # Sends the multisend transaction
+	
+	#stages the transaction with the gnosis api 
+	#note that transaction is automatically prompted for execution if enough signers are available in local
```
Comment (fubuloubu) [inline ape_safe/multisend.py:105]:
```suggestion
	# Stage the transaction to publish on-chain
	# NOTE: if not enough signers are available, publish to Safe API instead
```

**Review by fubuloubu — APPROVED** [review]

**Review by fubuloubu — APPROVED** [review]
Will merge after we complete detaching this from banteg's upstream

File: ape_safe/multisend.py [hunk 4]
```diff
@@ -90,13 +90,21 @@ class MultiSend(ManagerAccessMixin):
...
         txn.add(contract.myMethod, *call_args)
         txn.add(contract.myMethod, *call_args)
         ...  # Add as many calls as desired to execute
         txn.add(contract.myMethod, *call_args)
-        receipt = txn(sender=my_signer)  # Sends the multisend transaction
+	
+	# Stage the transaction to publish on-chain
+	# NOTE: if not enough signers are available, publish to Safe API instead
+        receipt = txn(sender=safe,gas=0)
+
```
Comment (fubuloubu) [inline ape_safe/multisend.py:107]:
```suggestion
```

**Pull request closed without merging**